source/advanced_components/ddc/reference/virtual/virtual_wait.rst

```diff
@@ -0,0 +1,60 @@
+.. _ddc_virtual_wait:
+
+============
+Wait Objects
+============
+
+Some objects take a bit to be ready, concretely images depending on the size. To manage this, you can use the attribute wait, it can have two possible values:
+
+- **false**: just create the objects and continue.
+- **true**: create objects and wait until they are successfully imported.
+
+For example:
+
+.. code::
+
+    images:
+      - name: "test_image"
+        ds_id: 1
+        size: 2048
+        meta:
+          wait: false
+          mode: 644
+
+In this example, the image would be created and there will not be any wait until it is ready, the program would continue.
+
+The timeout to wait until the resource is ready is also configurable, it can be done adding **wait_timeout** attribute in the object. For example:
+
+.. code::
+
+    images:
+      - name: "test_image"
+        ds_id: 1
+        size: 2048
+        meta:
+          wait: true
+          wait_timeout: 30
+
+In this example, the timeout to wait would be 30 seconds.
+
+.. note:: Wait is also used when deleting the objects, so if it is true the program will wait until the object is deleted.
```

Comment:
Doesn't make sense to have this exception for images on delete.
On delete we wait for all objects to be gone (e.g., esp VMs).